docker-compose.e2e.yml

```diff
@@ -0,0 +1,156 @@
+      - '9090:9090'
+    environment:
+      - MINIO_ROOT_USER=minioadmin
+      - MINIO_ROOT_PASSWORD=minioadmin
+    volumes:
+      - ./minio_data/identity-resolver-service-object-store:/data
+
+  mock-global-gs1-resolver:
+    # Replace with local or deployed image
+    image: ghcr.io/pyx-industries/pyx-identity-resolver:sha-fa017fb-local@sha256:cd514863bdaf65ab1b47b8e71669a0245b041c861285a83fa7cf0a65789d4e58
+    ports:
+      - '3001:3001'
+    environment:
+      - OBJECT_STORAGE_ENDPOINT=mock-global-gs1-resolver-object-store
+      - OBJECT_STORAGE_PORT=9000
+      - OBJECT_STORAGE_USE_SSL=false
+      - OBJECT_STORAGE_ACCESS_KEY=minioadmin
+      - OBJECT_STORAGE_SECRET_KEY=minioadmin
+      - OBJECT_STORAGE_BUCKET_NAME=idr-bucket-2
+      - OBJECT_STORAGE_PATH_STYLE=true
+      - IDENTIFIER_PATH=identifiers
+      - API_KEY=test456
+      - APP_ENDPOINT=http://localhost:3001
+      - APP_NAME=IDR-2
+      - RESOLVER_DOMAIN=http://localhost:3001
+      - LINK_TYPE_VOC_DOMAIN=http://localhost:3001/voc
+      - PORT=3001
+    depends_on:
+      - mock-global-gs1-resolver-object-store
+
+  mock-global-gs1-resolver-object-store:
+    image: quay.io/minio/minio:RELEASE.2024-08-17T01-24-54Z-cpuv1
+    command: server /data --console-address ":9091"
+    ports:
+      - '9001:9000'
+      - '9091:9090'
+    environment:
+      - MINIO_ROOT_USER=minioadmin
+      - MINIO_ROOT_PASSWORD=minioadmin
+    volumes:
+      - ./minio_data/mock-global-gs1-resolver-object-store:/data
+
+  seeding-data:
+    image: alpine:3.20.3
+    environment:
+      # IDR service environment variables
+      - IDR_SERVICE_HOST=identity-resolver-service
+      - IDR_SERVICE_PORT=3000
+      - IDR_SERVICE_API_KEY=test123
+
+      # Mock GS1 service environment variables
+      - IDR_SERVICE_DOMAIN=http://localhost:3000
+      - MOCK_GS1_SERVICE_HOST=mock-global-gs1-resolver
+      - MOCK_GS1_SERVICE_PORT=3001
+      - MOCK_GS1_SERVICE_API_KEY=test456
+    command:
+      - /bin/sh
+      - -c
+      - |
+        if [ "$SEEDING" = "true" ]; then
+          apk add curl jq && \
+          ./seeding/idr-data.sh
+          ./seeding/mock-gs1-data.sh
+        fi
+    volumes:
+      - ./seeding:/seeding
+    depends_on:
+      - identity-resolver-service
+      - mock-global-gs1-resolver
+  # Mock App
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        CONFIG_FILE: app-config.e2e.json
+    ports:
+      - "3003:80"
+
+  cypress:
+    image: cypress/included:13.16.1
+    depends_on:
+      - app
+    environment:
+      - CYPRESS_baseUrl=http://app:3003 # Base URL for Cypress to test
+    volumes:
+      - ./cypress:/e2e/cypress # Mount the Cypress test files
+      - ./cypress.config.ts:/e2e/cypress.config.ts # Mount Cypress config file
+      - ./packages/mock-app:/app/packages/mock-app # Mount the mock-app for context
+    working_dir: /e2e
+    entrypoint: ["sh", "-c", "npx wait-on http://app:3003 && npx cypress run"]
```

Comment:
I'm not sure if this setup, including both the configuration and requirements, is correct.
I believe we only need to set up docker-compose for VCkit, Storage Service, IDR, and Mock GS1 services.

I’ve read your usage instructions in the README.md for running E2E testing and noticed that we manually run the mock-app and Cypress without docker-compose.

Therefore, I think this setup might not be necessary.